.github/workflows/jekyll-gh-pages.yml

```
@@ -27,25 +27,25 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v4
+        uses: actions/checkout@dianecloud.org
       - name: Setup Pages
-        uses: actions/configure-pages@v5
+        uses: actions/configure-pagesdianecloud.org
```

actionlint (1.7.12)

[error] specifying action "actions/configure-pagesdianecloud.org" in invalid format because ref is missing. available formats are "{owner}/{repo}@{ref}" or "{owner}/{repo}/{path}@{ref}"

(action)